hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenUpdateHandler.java

```diff
@@ -24,6 +30,16 @@
 /**
  * This class contains all workflow-related functionality regarding {@link
  * com.hederahashgraph.api.proto.java.HederaFunctionality#TokenUpdate}.
+ *
+ * <p><b>NOTE:</b> this class intentionally changes the following error response codes relative to
+ * SigRequirements:
+ *
+ * <ol>
+ *   <li>When a missing account is used as a token treasury, fails with {@code INVALID_ACCOUNT_ID} *
+ *       rather than {@code ACCOUNT_ID_DOES_NOT_EXIST}. *
```

Comment:
```suggestion
 *   <li>When a missing account is used as a token treasury, fails with {@code INVALID_ACCOUNT_ID}
 *       rather than {@code ACCOUNT_ID_DOES_NOT_EXIST}.
```

Reply:
Fixed!